src/main/java/org/kairosdb/datastore/cassandra/CassandraConfiguration.java

```diff
@@ -56,6 +56,9 @@
 	public static final String CONNECTION_TIMEOUT_PROPERTY = "kairosdb.datastore.cassandra.connection_timeout";
 	public static final String READ_TIMEOUT_PROPERTY = "kairosdb.datastore.cassandra.read_timeout";
 
+	public static final boolean DEFAULT_ALIGN_DATAPOINT_TTL_WITH_TIMESTAMP = false;
+	public static final int DEFAULT_DATAPOINT_TTL = 0; //Zero ttl means data lives forever.
+	public static final boolean DEFAULT_FORCE_DEFAULT_DATAPOINT_TTL = false;
```

Comment:
Just a note that if we are able to inject CassandraConfiguration directly that these changes become unnecessary.